addons/actions/package.json

```diff
@@ -73,6 +92,13 @@
   "publishConfig": {
     "access": "public"
   },
+  "bundler": {
+    "entries": [
+      "./src/index.ts",
+      "./src/manager.tsx",
+      "./src/preset/preview.tsx"
```

Comment:
Please do not have a nested directory structure in entries.

Entrypoints should be directly in the `src` directory.